docs/packages/TypeScript_ESTree.mdx

```diff
@@ -33,6 +33,12 @@ Parses the given string of code with the options provided and returns an ESTree-
 
 ```ts
 interface ParseOptions {
+  /**
+   * Specify the `sourceType`.
+   * For more details, see https://github.com/typescript-eslint/typescript-eslint/pull/9121
```

Comment:
PR number changed

Reply:
> PR number changed

I accidentally opened a new PR, but most of the conversation about the problem and how to fix it happened in the old PR. The current PR is mostly about supplementing the documentation for the change, and updating the branch to match main. Since I mentioned the current PR as the last comment in the old PR, I thought it was appropriate to mention the old PR in the comments, because after all, what I want to show in the comments is why the property was needed, and that's also in the conversation in the old PR.

Should I change the PR number to the current PR?